client/app/telecom/orders/telecom-orders.html

```diff
@@ -10,78 +10,35 @@ <h1 data-translate="telecom_orders_follow_up_title"></h1>
-                            </td>
-                            <td data-ng-bind="'telecom_orders_follow_up_type_' + order.accessType | translate"></td>
-                            <td data-ng-bind="'status_' + order.lineStatus | translate"></td>
-                            <td>
-                                <span data-ng-if="order.lastTask"
-                                      data-ng-bind="'telecom_common_pack_xdsl_order_step_name_' + order.lastTask.name | translate">
-                                </span>
-                            </td>
-                            <td>
-                                <i class="ovh-font ovh-font-1x4 my-0 text-{{ order.badgeClass }} {{ order.badgeIconClass }}"
-                                   data-uib-tooltip="{{ 'telecom_common_pack_xdsl_order_status_' + order.lastTask.status | translate }}"
-                                   data-tooltip-append-to-body="true">
-                                </i>
-                            </td>
-                        </tr>
-                        <tr data-ng-show="OrdersCtrl.loading">
-                            <td colspan="5">
-                                <oui-spinner></oui-spinner>
-                            </td>
-                        </tr>
-                        <tr data-ng-if="OrdersCtrl.ordersDetails.length == 0 && !OrdersCtrl.loading">
-                            <td colspan="5"
-                                class="text-center"
-                                data-translate="telecom_orders_follow_up_no_orders">
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
-            </div>
-            <div class="mt-2"
-                 data-ng-show="OrdersCtrl.orders.length > OrdersCtrl.perPage"
-                 data-pagination-front
-                 data-go-to-page="false"
-                 data-items="OrdersCtrl.orders"
-                 data-paginated-items="OrdersCtrl.ordersDetails"
-                 data-current-page="OrdersCtrl.currentPage"
-                 data-items-per-page="OrdersCtrl.perPage"
-                 data-nb-pages="OrdersCtrl.nbPages"
-                 data-on-transform-item-notify="OrdersCtrl.onTransformItemNotify(item)"
-                 data-page-placeholder="{{currentPage}} / {{nbPages}}"
-                 data-item-per-page-placeholder="{{'telecom_orders_follow_up_items_per_view' | translate}}">
-            </div>
+
+            <oui-datagrid data-rows="OrdersCtrl.ordersDetails">
+                <oui-column title="'telecom_orders_follow_up_header_service' | translate" data-property="name">
+                    <a data-ui-sref="telecom.orders.show({serviceName: $row.xdsl })"
+                        data-ng-bind="$row.name | limitTo: 25">
+                    </a>
+                </oui-column>
+                <oui-column title="'telecom_orders_follow_up_header_type' | translate" data-property="accessType">
```

Comment:
I'm using data-title, for unity purposes I think it would be better (there's like 5 of them, I din't not multiple comments)